### PR TITLE
DLS-4229: Play 2.6 library upgrades.

### PR DESCRIPTION
**Pull request by rambabu-posa**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -64,13 +56,16 @@ lazy val microservice = Project(appName, file("."))
...
+    Test / scalacOptions --= Seq("-Ywarn-value-discard")
   )
   .settings(scalacOptions ++= Seq("-Yrangepos", "-Ywarn-unused:imports"))
   .settings(publishingSettings: _*)
   .configs(IntegrationTest)
   .settings(integrationTestSettings(): _*)
-  .settings(resolvers ++= Seq(Resolver.jcenterRepo, "emueller-bintray" at "http://dl.bintray.com/emueller/maven"))
+  .settings(resolvers ++= Seq(
+    Resolver.jcenterRepo,
+    ("emueller-bintray" at "http://dl.bintray.com/emueller/maven").withAllowInsecureProtocol(true)
```
Comment (calinursaciuc) [inline build.sbt:67]: can we use https and remove the allow thing? Or they don't have it?

Comment (rambabu-posa) [reply build.sbt]: Am not able to use https here. Getting lots of compilation issues.

**Review by calinursaciuc — APPROVED** [review]
Make sure ui tests run fine

**Comment by rambabu-posa** [comment]
@calinursaciuc UI tests passed locally. Am going to merge this PR and run UI tests on Jenkins.

**Pull request merged**